mangaki/mangaki/utils/anidb.py

```diff
@@ -502,3 +502,38 @@ def get_or_update_work(self,
     getattr(settings, 'ANIDB_CLIENT', None),
     getattr(settings, 'ANIDB_VERSION', None)
 )
+
+def diff_between_anidb_and_local_tags(work: Work,
+                                      anidb_tags: Dict[str, Dict[str, Any]]) -> Dict[str, Dict[str, Dict[str, Any]]]:
```

Comment:
You can write type alias to improve readability, if you feel like:
```python
Tag = Dict[str, Any]

blablabla
-> Dict[str, Dict[str, Tag]]
```

But IMHO, at this point, we may enjoy the cost of a `typing.NamedTuple`.